pkgs/development/tools/misc/luarocks/default.nix

```diff
@@ -63,7 +65,7 @@ stdenv.mkDerivation rec {
     installShellCompletion --cmd luarocks --zsh <($out/bin/luarocks completion zsh)
   '';
 
-  propagatedBuildInputs = [ zip unzip cmake ];
+  propagatedNativeBuildInputs = [ zip unzip cmake ];
```

Comment:
Do we really need to do this?

Reply:
i don't know,  they were added in https://github.com/NixOS/nixpkgs/pull/54454 also out of scope of this pr

Reply:
@teto Do you still remember why this was added?